decidim-core/app/cells/decidim/user_conversation/show.erb

```diff
@@ -10,7 +10,9 @@
         <% end %>
       </div>
 
-      <% if conversation.accept_user?(user) %>
+      <% if conversation.with_deleted_users?(user) %>
+        <em><%= t "decidim.user_conversations.show.deleted_accounts" %></em>
```

Comment:
I know that you're being consistent with the `.not_allowed` styling, but I didn't see this `<em>` styling before. How about if we fix it for once? 

```suggestion
        <div class="callout warning margin-top-2"><%= t "decidim.user_conversations.show.deleted_accounts" %></div>
```